include/llvm/Support/WinMacros.h

```diff
@@ -51,15 +51,28 @@
 #define UIntToInt(uint_arg, int_ptr_arg) *int_ptr_arg = uint_arg
 
 #define INVALID_HANDLE_VALUE ((HANDLE)(LONG_PTR)-1)
-#define ERROR_SUCCESS 0L
-#define ERROR_OUT_OF_STRUCTURES 0L
-#define ERROR_UNHANDLED_EXCEPTION 574L
-#define ERROR_NOT_FOUND 1168L
-#define ERROR_NOT_CAPABLE 775L
-#define ERROR_FILE_NOT_FOUND 2L
 
-// Note: This will nullify some functionality.
-#define SetLastError(err)
+// Use errno to implement {Get|Set}LastError
+#define GetLastError() errno
+#define SetLastError(ERR) errno = ERR
+#define GetWin32ErrorMessage(err) std::strerror(err)
```

Comment:
GetWIn32ErrorMessage is implemented for Windows as a static function in dxcapi.use.cpp that is currently ifdefed out. It might be more consistent to put the implementation there.

Reply:
makes sense. Will do.